app/questionnaire/questionnaire_schema.py

```diff
@@ -176,6 +180,11 @@ def _parse_schema(self):
         self._answers_by_id = self._get_answers_by_id()
         self.error_messages = self._get_error_messages()
 
+    def _get_section_id_for_list_block(self, block):
```

Comment:
I can see any usages for this?

Reply:
It's used on line 38 - `section_id = self._get_section_id_for_list_block(block)`

Reply:
Weird, IDE must have been lazy.